.github/workflows/post-release-changes.yml

```diff
@@ -28,16 +51,39 @@ jobs:
         with:
           repository: ${{ inputs.REPO_URL }}
           ref: ${{ inputs.REPO_BRANCH }}
+
       - name: Setup branch and env
         run: |
           # Strip git ref prefix from version
           echo "BRANCH_NAME=$(echo ${{ github.ref }} | sed -e 's,.*/\(.*\),\1,')" >> $GITHUB_ENV
           echo "GPG_TTY=$(tty)" >> $GITHUB_ENV
 
-      - name: Updating the OSSRH_SNAPSHOT_URL in push-trigger
+      - name: Debug Inputs
+        run: |
+          echo "UPDATE_RELEASE_URL: ${{ inputs.UPDATE_RELEASE_URL }}"
+          echo "UPDATE_SNAPSHOT_URL: ${{ inputs.UPDATE_SNAPSHOT_URL }}"
+          echo "UPDATE_POM_VERSION: ${{ inputs.UPDATE_POM_VERSION }}"
+          echo "FROM_VERSION: ${{ inputs.FROM_VERSION }}"
+          echo "TO_VERSION: ${{ inputs.TO_VERSION }}"
+
```

Comment:
add one more check to make sure both the UPDATE_RELEASE_URL and UPDATE_SNAPSHOT_URL is not having value as true.

Reply:
in case both the values are true fail with error